src/resources/elements/primeDesignSystem/_colors.scss

```diff
@@ -5,13 +5,15 @@ $Secondary02: #a258a7; // Accent color
 $Secondary03: #fff619; // Accent color
 $Secondary04: #ffce3d; // Accent color
 $Secondary05: #64557d; // Accent color
+$Purple-100: #64557d; // synonym for Secondary05
```

Comment:
why were these synonym added? I noticed the Figma design using these names but maybe they are old names. Can you confirm with Jaap if that's the case and use the "Secondary05" instead? If not, the best thing we can do is to assign the "Secondary05" variable to "$Purple-100": `$Purple-100: $Secondary05`

Reply:
Removed the synonyms because I think is best not to star tusing them yet.